shared/actions/fs/platform-specific.desktop.js

```diff
@@ -339,9 +341,6 @@ const loadUserFileEdits = (state: TypedState, action) =>
         )
         .toArray()
       yield Saga.sequentially([
-        // TODO (songgao): make a new action that accepts an array of updates,
-        // so that we only need to trigger one update through store/rpc/widget
-        // for all these each time.
```

Comment:
Did you mean to remove this? It seemed like a good idea to merge all these actions into one and deal with them once in the reducer.

Reply:
Nope, didn't mean to remove it. No idea how that happened...